etc/kayobe/ansible/roles/opensearch/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- import_tasks: set_log_retention_policy.yml
+  when:
+    - kolla_enable_central_logging | bool
```

Comment:
Would this be useful when central logging is disabled but opensearch is not?

Reply:
Hmm, the only case I could think of would be if CloudKitty was enabled, had been patched to use daily indices, and central logging was off. I think it's very unlikely. Maybe I missed another use case?

There is another corner case to consider if an external ES/OS cluster is being used. I.e. you could have `enable_opensearch: False` and `enable_central_logging: True`. I think in that case, you would still likely want a retention policy.